docker-compose.yml

```diff
@@ -41,6 +41,28 @@ services:
     volumes:
       - .:/usr/src/app
 
+  worker:
```

Comment:
Since this is just based off the same Dockerfile anyway, is it possible to combine this into the main api container?

Reply:
Is there a benefit to combining it with the main API container? I thought part of the point of Docker was so that it's really easy to create separate containers for each process.

If you're just talking about the repetition of the configuration between the `api` and `worker` services in docker-compose.yml, rather than the actual run-time of the two processes, we could make use of YAML's syntax to define a hashmap/object with the common attributes and include them into both the `api` and `worker` service. See the examples in https://docs.docker.com/compose/compose-file/#extension-fields

Reply:
Yea, in this instance I feel running/managing an additional container to run a cron is under the threshold of value for the extra overheard. It's a balance, but in this case I would lean towards simplicity.

Reply:
@richardxia Sounds great. I agree - I think it might be best to keep the `api` and `worker` containers separate like they currently are in this PR after all. Nevertheless, given my unfamiliarity with the GCP devops infrastructure, I'm also open to combining them if it's needed.

Now that the `docker-compose.yml` file has been refactored, is it ok to merge? I'll spend some time looking at the gcloud repo to see if I can figure out which changes are needed in the deployment setup.